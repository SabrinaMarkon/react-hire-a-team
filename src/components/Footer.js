import React from 'react';

export default function Footer() {

  return (

    <footer>
      Made with Love 4 U by Sabrina
    </footer>

  )
}