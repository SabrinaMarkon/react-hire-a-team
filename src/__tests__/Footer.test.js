import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';

// components
import Footer from '../components/Footer';

function setup() {
  const wrapper = shallow(<Footer />);
  return wrapper;
}

describe('Testing footer component', () => {
  it('renders the footer', () => {
    const footer = document.createElement('footer');
    ReactDOM.render(<Footer />, footer);
    ReactDOM.unmountComponentAtNode(footer);
  });
});