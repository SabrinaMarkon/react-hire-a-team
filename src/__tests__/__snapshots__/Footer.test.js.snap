// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing footer component it should match the Footer snapshot 1`] = `
<footer>
  Made with Love 4 U by Sabrina
</footer>
`;
